qprofiler2/src/org/qcmg/qprofiler2/QProfiler2.java

```diff
@@ -70,10 +71,10 @@
 	 * method is invoked so we can assume that the class fields are populated
 	 * and ready for us to use.
 	 */
-	protected int engage() throws Exception {
+	protected int engage() throws Exception { 
 		Element root = XmlElementUtils.createRootElement( "qProfiler", null);
 		
-		// Create new Summary object ready to hold our processing
+		 //  Create new Summary object ready to hold our processing
```

Comment:
same in this file - a whitespace has been added before and after _every_ occurrence of `//` where for most of them it should only have been added after.